### PR TITLE
Improve overall mobile responsiveness + Design improvements

### PR DESCRIPTION
**Pull request by Cold-A-Muse**

### CODE REVIEW

File: packages/web/src/components/molecules/SupplySummaly/index.tsx [hunk 1]
```diff
@@ -69,15 +69,15 @@ export const SupplySummary = ({ apy, creators, annualSupplyGrowthRatio }: Props)
...
         </SupplyBadge>
         <SupplyBadge>
           <Statistics>{creators?.dp(2).toNumber()}%</Statistics>
-          <span>APY for Creators</span>
+          <span>APY Creators</span>
         </SupplyBadge>
         <SupplyBadge>
           <Statistics>{annualSupplyGrowthRatio?.dp(2).toNumber()}%</Statistics>
-          <span>ASG of DEV</span>
+          <span>ASG DEV</span>
```
Comment (hhatto) [inline packages/web/src/components/molecules/SupplySummaly/index.tsx:80]: https://www.figma.com/file/2XztNCtx3iXyPECgnNtIAZ/Stakes-Social.-0.1?node-id=360%3A0
It differs from the wording in Figma.
Is this OK?

see also: https://github.com/dev-protocol/stakes.social/pull/577

**Review by hhatto — COMMENTED** [review]
I've checked the operation and it's very good 👍 
commented on a couple of minor points, if you can confirm them.

File: packages/web/src/components/molecules/SupplySummaly/index.tsx (hunk 1 above)
Comment (defi-er) [reply packages/web/src/components/molecules/SupplySummaly/index.tsx:80]: APY Stakers is fine @hhatto

**Comment by Cold-A-Muse** [comment]
@hhatto I have implemented your suggested changes :)!

**Review by hhatto — APPROVED** [review]
@Cold-A-Muse Thanks!! LGTM

**Pull request merged**